### ISSUE TITLE
[Suggesetion] Can mystical agriculture come back to AOF7?

### ISSUE THREAD

**Issue by StevenUST** [TeamAOF/All-of-Fabric-7#108]
I am interested in playing AOF starts from AOF4 until now. However, when I play AOF6, I found mystical agriculture is a really powerful mod that helps me build the metal farm (combine with Create and Modern Industrialization). Can this mod come back to AOF7 as I have found that the fabric version of this mod has been released.

The link to download this mod is:
https://www.curseforge.com/minecraft/mc-mods/mystical-agriculture-refabricated

**Comment by RainbowIris323** [comment]
In my opinion I thought it was a "bit" over powered and allowed for skiping / fast tracking many questlines with just a few farms

**Comment by Kazuhiko-Gushiken** [comment]
I agree, it was way too OP. We even had to remove the metal type plants because it would kill the progression difficulty of some tech mods.
